businessRules/donations.js

```diff
@@ -1,54 +1,64 @@
 const { v4 } = require('uuid');
 const { createDonation } = require('../models/donations');
-const { getAllIdeas } = require('../models/ideas');
-const { getIdeas } = require('../utils/storage');
+const { validateIdea } = require('../validations/ideas');
+const { validateSum } = require('../validations/donations.js');
+const { setUsers, getUsers } = require('../utils/storage');
+const { getUser } = require('../models/users');
+const { getIdea } = require('../models/ideas');
 
-function createDonation(firstName, sum, userId, ideaId) {
-    let ideas = getIdeas();
-    const idea = ideas.find((idea) => ideaId === idea.id);
+function createAnonymousUser(firstName) {
+    let users = getUsers();
 
-    if (!idea) {
-        throw new Error('Error. No idea with such ID found.');
-    }
-
-    const generateId = v4();
-    const anonymous = 'Anonymous';
+    const userId = v4();
+    const defaultFirstName = 'Anonymous';
+    const defaultUsername = 'none';
```

Comment:
Instead of `none`, can it simply be `null`?

Reply:
Oh, perhaps. Fixed that.